python_modules/dagster/dagster/_core/instance/config.py

```diff
@@ -325,4 +325,9 @@ def dagster_instance_config_schema() -> Mapping[str, Field]:
         "retention": retention_config_schema(),
         "sensors": sensors_daemon_config(),
         "schedules": schedules_daemon_config(),
+        "auto_materialize": Field(
+            {
+                "enabled": Field(Bool, is_required=False),
```

Comment:
we don't necessarily have to publicize this field since it defaults to on, but it gives an escape hatch